ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/response/JsonRpcError.java

```diff
@@ -115,10 +115,15 @@
   VALUE_NOT_ZERO(-50100, "We cannot transfer ether in private transaction yet."),
   DECODE_ERROR(-50100, "Unable to decode the private signed raw transaction"),
   GET_PRIVATE_TRANSACTION_NONCE_ERROR(-50100, "Unable to determine nonce for account in group."),
-  PRIVACY_GROUP_DOES_NOT_EXIST(-50100, "Privacy group does not exist."),
+  OFFCHAIN_PRIVACY_GROUP_DOES_NOT_EXIST(-50100, "OffChain Privacy group does not exist."),
+  ONCCHAIN_PRIVACY_GROUP_DOES_NOT_EXIST(-50100, "OnChain Privacy group does not exist."),
```

Comment:
I believe `Offchain` and `Onchain` are the spelling we use across the board.

Reply:
Done